docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -280,28 +302,80 @@ certoraRun ... --prover_args '-dontStopAtFirstSplitTimeout true -depth 15 -mediu
 ### Dealing with nonlinear arithmetic
 
 Nonlinear integer arithmetic is often the hardest part of the formulas that
-Certora Prover is solving.
+Certora Prover is solving. 
+
+The Certora Prover displays the absolute number of nonlinear operations, as well
+as their number per external call, in the Live Statistics panel. In the per-call
+display, there is a warning-sign next to the call when there is a non-trivial
```

Comment:
this number includes nonlinear operations in calls from this function, right? both internal and externall

Reply:
yeah, everything below this function is counted
so as if everything was inlined (should state this clearly), i.e., the numbers will always descend when going deeper into the tree
 --> will try to clarify